public/.well-known/openid-credential-issuer

```diff
@@ -1,76 +1,84 @@
 {
-    "credential_issuer": "https://credential-issuer.example.com",
-    "authorization_servers": [ "https://server.example.com" ],
-    "credential_endpoint": "https://credential-issuer.example.com",
-    "batch_credential_endpoint": "https://credential-issuer.example.com/batch_credential",
-    "deferred_credential_endpoint": "https://credential-issuer.example.com/deferred_credential",
-    "display": [
-        {
-            "name": "Example University",
-            "locale": "en-US"
-        },
-        {
-            "name": "Example Université",
-            "locale": "fr-FR"
-        }
+    "authorization_endpoint": "https://authz-server1.zenswarm.forkbomb.eu/authorize",
+    "authorization_response_iss_parameter_supported": true,
+    "authorization_servers": [
+        "https://authz-server1.zenswarm.forkbomb.eu/"
+    ],
+    "batch_credential_endpoint": "https://issuer1.zenswarm.forkbomb.eu/batch_credential",
+    "claim_types_supported": [
+        "normal"
     ],
+    "claims_parameter_supported": false,
+    "code_challenge_methods_supported": [
+        "S256"
+    ],
+    "credential_endpoint": "https://issuer1.zenswarm.forkbomb.eu/credential",
+    "credential_issuer": "https://issuer1.zenswarm.forkbomb.eu/",
     "credentials_supported": {
-        "UniversityDegreeCredential": {
-            "format": "jwt_vc_json",
-            "scope": "UniversityDegree",
+        "Auth1": {
```

Comment:
This list is populated obviously from the signroom. @bbtgnn please reflect this changes correctly ;)